Sources/Public/CalendarView.swift

```diff
@@ -247,7 +247,7 @@ public final class CalendarView: UIView {
     cachedAccessibilityElements = nil
     if let element = focusedAccessibilityElement as? OffScreenCalendarItemAccessibilityElement {
       UIAccessibility.post(
-        notification: .layoutChanged,
+        notification: .screenChanged,
         argument: visibleViewsForVisibleItems[element.correspondingItem])
```

Comment:
I'm confused by the two paths we have to focus items. This one in `layoutSubviews` is for visible views but we check if it conforms to `OffScreenCalendarItemAccessibilityElement` first?

And then why are we focusing separately in the configure function?